Add "Translating your form" article

kariannemah opened the pull request:
Closes #333. An 85% done WIP that I'll finish up on Monday.

I need to spruce up the copy and add a couple of screenshots for respondent view.

@ajb, do we have an `app_pages` value for the Translations settings page yet? I couldn't find one in the `screendoor-v2` codebase. Right now the only one I've set up is for General settings, `projectadmin-settings-setting`.

ajb commented:
> @ajb, do we have an app_pages value for the Translations settings page yet? I couldn't find one in the screendoor-v2 codebase. Right now the only one I've set up is for General settings, projectadmin-settings-setting.

Looks like it's `projectadmin-languages-index`. (Here's how to find it: http://take.ms/xnLXX)

kariannemah commented:
Thanks! I changed `projectadmin-settings-setting` to `projectadmin-languages-index` and added `projectadmin-wizard-responseform` as another app page.

I call [the info box that contains the URL for a language's translation page](http://take.ms/tThF8) a "translation module." @ajb, I'm totally fine renaming it if you think that's confusing.

This is ready for review.

ajb commented on contents/articles/screendoor/your_form/translating_your_form.md:
```diff
@@ -0,0 +1,81 @@
+
+### Can I notify respondents of status changes in a translated project?
+
+Since status cannot be translated, we do not support automatic of status change notification in translated projects.
```
Looks like a typo in here

kariannemah commented:
I went with "element." Also fixed those two typos.

ajb commented:
Sweet, thanks!
